moto/emr/models.py

```diff
@@ -63,6 +71,34 @@ def __init__(
     def set_instance_count(self, instance_count):
         self.num_instances = instance_count
 
+    @property
+    def auto_scaling_policy(self):
+        return self._auto_scaling_policy
+
+    @auto_scaling_policy.setter
+    def auto_scaling_policy(self, value):
+        if value is None:
+            self._auto_scaling_policy = value
+            return
+        self._auto_scaling_policy = CamelToUnderscoresWalker.parse(value)
```

Comment:
Here is the generic Camel to underscore conversion of the deep AutoScalingPolicy parameter.